gits/terminal.py

```diff
@@ -213,46 +252,66 @@ def _default_rendition(self):
         """
         self._cap_set_color(0)
 
+    def _set_bg_color(self, color):
+        """Sets background color. """
+        color_bits, _ = divmod(self._sgr, MAGIC_NUMBER)
+        _, fg = divmod(color_bits, 16)
+        new_color_bits = color * 16 + fg
+        self._sgr &= ~(color_bits << 40)  # clear color bits
```

Comment:
Could we replace this bits manipulation with something similar to `_set_bit` or `_clean_bit`. 
E.g.
```python
def _set_bits(self, src, bit):
    self._sgr |= src << bit
```
Usage:
```python
self._set_bits(color_bits, 40)
```